atc/worker/placement.go

```diff
@@ -175,12 +175,25 @@ func (strategy volumeLocalityStrategy) Order(logger lager.Logger, pool Pool, wor
 			_, found, err := pool.db.VolumeRepo.FindResourceCacheVolume(worker.Name(), resourceCache)
 			if err != nil {
 				logger.Error("failed-to-find-resource-cache-volume", err)
-				return nil, err
+				return err
 			}
 			if found {
 				counts[worker.Name()]++
 			}
 		}
+		return nil
+	}
+
+	err := updateCountsForArtifact(spec.ImageSpec.ImageArtifact, "/")
```

Comment:
Usually image is much than other artifacts, can we give image 5 counts?

That way will especially benefit those use case where some job uses huge task images, but the job doesn't run frequently. Current image-get runs on a worker, task runs on the other worker, image streaming take longer time than directly pulling image from registry, because volume streaming involves tar->gzip->ungzip->untar.